nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -9,6 +9,30 @@
 logger = logging.getLogger(__name__)
 
 
+def add_location_contenttype_to_site_status_and_tags(apps, site_ct, location_ct):
+    """
+    Ensure that the Statuses and Tags assigned to Site can also be assigned to Locations
+    before creating Site LocationType locations.
+
+    Args:
+        apps: Installed apps
+        site_ct: ContentType for Site Model Class
+        location_ct: ContentType for Location Model Class
+    """
+    # Status
+    Status = apps.get_model("extras", "status")
+    statuses = Status.objects.filter(content_types__in=[site_ct])
+    for status in statuses:
+        status.content_types.add(location_ct)
+        status.save()
```

Comment:
Not a huge deal but you don't need to call `save()` when updating an m2m, since that's updating the through table and not the model's table.